db/migrate/20220624211208_delete_all_comments.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteAllComments < ActiveRecord::Migration[5.0]
+  def up
+    execute <<~SQL
+      DELETE FROM messages WHERE "type" = 'Comment';
```

Comment:
Should "type" be in double-quotes? Just checking.

Reply:
I tried a few variants but the main thing I'm guarding against here is the fact that "type" is an SQL keyword. So I need it to be escaped somehow by Rails. Hence the unorthodox quoting style.

Reply:
Ah, I believe this case they usually escape with backticks, no?

Reply:
Hmm maybe. Let me try this now.

Reply:
Syntax error:

```
magne@C02ZV1LNMD6W excelsior % bundle exec rake db:migrate 
/Users/magne/src/excelsior/config/initializers/devise.rb:227: warning: already initialized constant OpenSSL::SSL::VERIFY_PEER
== 20220624211208 DeleteAllComments: migrating ================================
-- execute("DELETE FROM messages WHERE `type` = 'Comment';\n")
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::UndefinedFunction: ERROR:  operator does not exist: ` character varying
LINE 1: DELETE FROM messages WHERE `type` = 'Comment';
                                   ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
: DELETE FROM messages WHERE `type` = 'Comment';
```

Reply:
Hmm...Ah...seems like it is using PostGRES...MySQL and BigQuery's Standard SQL support the backticks, but looks like PG doesn't.